flask_monitoringdashboard/core/config/__init__.py

```diff
@@ -27,7 +27,7 @@ def __init__(self):
         self.link = 'dashboard'
         self.monitor_level = 1
         self.outlier_detection_constant = 2.5
-        self.sampling_period = 0
+        self.sampling_period = 5/1000.0
```

Comment:
doesn't work in python 2.7, as it evaluates to 0

use the following import:
`from __future__ import division`

Reply:
1000.0 is a float so the result will be a float, even in 2.7